source/simulator/core.cc

```diff
@@ -1682,6 +1682,17 @@ namespace aspect
                                          T_preconditioner);
           solve_advection(AdvectionField::temperature());
 
+          if (
+            parameters.use_discontinuous_temperature_discretization
+            &&
+            parameters.use_limiter_for_discontinuous_temperature_solution
+          )
+            {
+              pcout << "Apply bound preserving limiter to the temperature advection field "
```

Comment:
Ah now I understand why it is outside of the function. You want a single line instead of one per field. Then it is indeed better to leave it here. But please rephrase the output according to my comment below.